app.js

```diff
@@ -14,62 +17,82 @@ const catchphraseButton = document.getElementById('catchphrase-button');
 // set state for how many times the user changes the head, middle, and bottom
 // set state for all of the character's catchphrases
 
-headDropdown.addEventListener('change', () => {
-    // get the value of the head dropdown
+let headCount = 0;
+let middleCount = 0;
+let bottomCount = 0;
+let catchPhrases = [''];
 
+headDropdown.addEventListener('change', (e) => {
+     // get the value of the head dropdown
+    const value = e.target.value;
```

Comment:
Another way to grab this value is to call on the headDropdown input directly and pull the value off of it: 

```js
const value = headDropdown.value;
```